UltraStar Play/Assets/Common/Model/Song/Builder/SongMetaBuilder.cs

```diff
@@ -112,12 +112,23 @@ public static SongMeta ParseFile(string path, Encoding enc = null)
                 }
             }
 
+			//Read the song file body
+            string songBodyText = "";
+            string bodyLine;
+            while ((bodyLine = reader.ReadLine()) != null)
+                songBodyText += bodyLine; //Ignorning the newlines for the hash
```

Comment:
Please use curly braces always for control structures such as while, if, for, etc. - even for a single statement.